fv3core/tests/savepoint/translate/translate_haloupdate.py

```diff
@@ -150,30 +146,31 @@ def __init__(
         super(TranslateHaloVectorUpdate, self).__init__(grid, namelist, stencil_factory)
 
     def compute_parallel(self, inputs, communicator):
-        logger.debug(f"starting on {communicator.rank}")
+        pace_log.debug(f"starting on {communicator.rank}")
         state = self.state_from_inputs(inputs)
         req = communicator.start_vector_halo_update(
             state["x_wind_on_c_grid"], state["y_wind_on_c_grid"], n_points=utils.halo
         )
 
-        logger.debug(f"finishing on {communicator.rank}")
+        pace_log.debug(f"finishing on {communicator.rank}")
         req.wait()
         return self.outputs_from_state(state)
 
     def compute_sequential(self, inputs_list, communicator_list):
         state_list = self.state_list_from_inputs_list(inputs_list)
         req_list = []
         for state, communicator in zip(state_list, communicator_list):
-            logger.debug(f"starting on {communicator.rank}")
+            pace_log.debug(f"starting on {communicator.rank}")
             req_list.append(
                 communicator.start_vector_halo_update(
+                    state["x_wind_on_c_grid"].data.dtype,
```

Comment:
Can we do this implicitly instead of specifying the precision/type in the halo update call...

Reply:
nvm

Reply:
Yeah this is actually wrong... I forgot to push some test fix